docs/assemblies/proc_deploying-in-disconnected-environments.adoc

```diff
@@ -0,0 +1,50 @@
+[id="proc_deploying-in-disconnected-environments"]
+= Deploying OpenStack in a disconnected environment
+
+[role="_abstract"]
+== Process
+Deploying in disconnected environments can be achieved largely by following the OpenShift documentation for mirroring OLM Operators: https://docs.openshift.com/container-platform/4.16/installing/disconnected_install/installing-mirroring-installation-images.html#olm-mirror-catalog_installing-mirroring-installation-images
```

Comment:
I'd like to add an example of ImageSetConfiguration CR. Because the default ImageSetConfiguration CR won't include openstack-must-gather image and must-gather image for OCP. We should add those images in additionalImages option for debugging.

Reply:
Hey Keigo, thanks for taking a look. There is no additional step required by users to add the `openstack-must-gather` image. It is included in the `RelatedImages` for the `openstack-operator` and therefore gets mirrored by default. This is a snippet from the `ImageContentSourcePolicy` file that was generated in my lab after mirroring the operators:
~~~
[m3@osp-df-3 manifests-redhat-operator-index-1736121888]$ grep must-gather imageContentSourcePolicy.yaml
    - osp-df-3.lab.<redacted-domain>:8443/olm/rhoso-operators-openstack-must-gather-rhel9
    source: registry.redhat.io/rhoso-operators/openstack-must-gather-rhel9
~~~

This works because of the change I submitted here:
https://github.com/openstack-k8s-operators/openstack-operator/pull/1129

Reply:
Thanks Brendan. I didn't know the PR.  So, we don't need to add additionalImages separately.